.github/workflows/ci.yml

```diff
@@ -22,6 +23,21 @@ on:
   pull_request:
     branches:
       - main
+  # Support merge queues.
+  merge_group:
+    types:
+      - checks_requested
+  # Allow manual invocation – useful for debugging.
```

Comment:
manual invocation how?

Reply:
The comment refers to the next line, which is `workflow_dispatch`.

C.f. https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/manually-running-a-workflow